projects/addon-doc/components/api/api-item.component.ts

```diff
@@ -81,6 +87,21 @@ export class TuiDocAPIItem<T> implements OnInit {
         this.setQueryParam(value);
     }
 
+    public emitEvent(event: unknown): void {
+        // For more convenient debugging
+        console.info(this.name, event);
+
+        this.emits.update((x) => ++x);
+
+        let content: string | undefined;
+
+        if (event !== undefined) {
+            content = tuiInspectAny(event, 2);
+        }
+
+        this.alerts.open(content, {label: this.name}).subscribe();
```

Comment:
should we use `takeUntilDestroyed`?

Reply:
I remember there are nuances that in order for alerts to work correctly, they don't seem to need to be attached to the component's lifecycle, but I could be wrong.. @waterplea will it work correctly here with `takeUntilDestroyed` ?

Reply:
All alerts in doc are without `takeUntilDestroyed`
https://taiga-ui.dev/components/alert

Reply:
Yes, no need for it here